spec/services/section/new_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+RSpec.describe Section::NewService do
+  it 'returns the manual and a new section' do
+    user = double(:user)
+    section = double(:section)
+    manual = double(:manual, build_section: section)
```

Comment:
Having thought about it some more I've decided that I prefer the updated tests that avoid using `double`s. I've updated this branch so that it contains a single commit introducing the more realistic tests.